README.md

```diff
@@ -53,7 +53,22 @@ pip install -e .
 ```
 
 ## How to run
-The directory paths and parameters will need to be updated in the `config.tsv` file. 
+
+**Steps:**
```

Comment:
Maybe "Steps to identify generic DE genes on new data" or something like that would be a clearer heading?